docs/sdkx_ios/delegates.mdx

```diff
@@ -33,215 +33,229 @@ All the public APIs in the SDK should be called after initializing the SDK via [
-    Helpshift.sharedInstance.delegate = self;
-```
+Please make sure you set the delegate before calling the Conversation API so that they can callback your delegate methods.
 
-</TabItem>
-<TabItem value="Swift Example" label="Swift Example">
+## Helpshift Events {#helpshift-delegate-events}
 
-Class Implementation
+### Conversation Status Event {#conversationStatus}
 
-```swift
+This event contains information about the current ongoing conversation.
 
-    import Foundation
-    import HelpshiftX
+- Event name: `HelpshiftEventNameConversationStatus` <br />
+- Event Data:
+    - `HelpshiftEventDataLatestIssueId` <br />
+    - `HelpshiftEventDataLatestIssuePublishId` <br />
+    - `HelpshiftEventDataIsIssueOpen`
 
-    class HelpshiftDelegateHandler: HelpshiftDelegate {
-        func handleHelpshiftEvent(_ eventName: String, withData data: [AnyHashable : Any]?) {
-            // Your code here
-        }
-        func authenticationFailedForUser(with reason: HelpshiftAuthenticationFailureReason) {
-            // Your code here
-        }
-    }
+<Tabs groupId="ios-languages">
+
+<TabItem value="Objective-C" label="Objective-C">
 
+```objc
+- (void) handleHelpshiftEvent:(NSString *)eventName withData:(NSDictionary *)data {
+    ...
+    if([eventName isEqualToString:HelpshiftEventNameConversationStatus]) {
+        NSLog(@"Issue ID: %@", data[HelpshiftEventDataLatestIssueId]);
+        NSLog(@"Publish ID: %@", data[HelpshiftEventDataLatestIssuePublishId]);
+        NSLog(@"Is issue open: %@", data[HelpshiftEventDataIsIssueOpen]);
+    }
+}
 ```
 
-Class Usage:
+</TabItem>
+<TabItem value="Swift" label="Swift">
 
 ```swift
+func handleHelpshiftEvent(_ eventName: String, withData data: [AnyHashable : Any]?) {
+    if eventName == HelpshiftEventNameConversationStatus {
+        guard let data = data else { return }
+        let issueId = data[HelpshiftEventDataLatestIssueId]
+        let publishId = data[HelpshiftEventDataLatestIssuePublishId]
+        let issueOpen = data[HelpshiftEventDataIsIssueOpen]
 
-	var delegateHandler: HelpshiftDelegateHandler
-	.
-	.
-	.
-	.
-    delegateHandler = HelpshiftDelegateHandler()
-    Helpshift.sharedInstance().delegate =  delegateHandler
+        print("Issue id \(issueId), publishId: \(publishId), issueOpen: \(issueOpen)")
+    }
+}
 ```
 
 </TabItem>
 
 </Tabs>
 
-Please make sure you set the delegate before calling the Conversation API so that they can callback your delegate methods.
-
-## Helpshift Events {#helpshift-delegate-events}
-
-### Helpshift session started event {#session-delegates}
-
-This event gets fired when the Helpshift session starts
+### Widget Toggle Event {#widgetToggle}
 
-Event name: `HelpshiftEventNameSessionStarted` <br />
-Event data: `nil`
+This event is triggered when the user opens or exits the chat screen. This event is triggered with a boolean value of `"visible"` key. For your reference, see the below example:
 
-Example:
+- Event name: `HelpshiftEvent.WIDGET_TOGGLE`
```

Comment:
This is android events in iOS it could be, the below example code looks good
1. HelpshiftEventNameWidgetToggle
2. HelpshiftEventDataVisible